Atlas/android/app/build.gradle

```diff
@@ -69,6 +69,9 @@ dependencies {
     // nice progress bar and FAB
     compile 'com.dmitrymalkovich.android:progress-fab:1.6'
 
+    // imageview edit
+    compile 'com.squareup.picasso:picasso:2.5.2'
```

Comment:
We already use Glide for image downloading; can you remove Picasso and use Glide instead.